dev/com.ibm.ws.jsf.2.2_fat/fat/src/com/ibm/ws/jsf22/fat/tests/JSF22ComponentTesterTests.java

```diff
@@ -57,6 +65,11 @@ public class JSF22ComponentTesterTests {
     @Server("jsfTestServer2")
     public static LibertyServer jsfTestServer2;
 
+    @Rule
+    public BrowserWebDriverContainer<?> chrome = new BrowserWebDriverContainer<>(FATSuite.getChromeImage()).withCapabilities(new ChromeOptions())
+                    .withAccessToHost(true)
+                    .withLogConsumer(new SimpleLogConsumer(JSF22ComponentTesterTests.class, "selenium-driver"));
```

Comment:
This `selenium-driver` logging is verbose, but I'd like to keep it in here for now. I'll have it removed after the tests go through and SOE a few times.

Reply:
Can you open an issue so we don't forget?

Reply:
Ah I see it here: https://github.com/OpenLiberty/open-liberty/issues/25681